core-bundle/src/Resources/contao/modules/ModuleSearch.php

```diff
@@ -301,6 +301,12 @@ protected function compile()
 					{
 						$arrContext[] = ' ' . $strContext . ' ';
 					}
+
+					// Skip other terms if the total length is already reached
+					if (array_sum(array_map('mb_strlen', $arrContext)) >= $this->totalLength)
```

Comment:
`$this->totalLength` needs to be `$totalLength` I think

Reply:
Good catch! 🦊 See https://github.com/contao/contao/pull/2101